pkg/sql/queuefeed/reader.go

```diff
@@ -347,13 +353,15 @@ func (r *Reader) ConfirmReceipt(ctx context.Context) {
 	}()
 
 	// Persist the checkpoint if we have one.
+	// Since the rangefeed checkpoint represents the frontier (minimum timestamp)
```

Comment:
again using a frontier in the rangefeed is the right solution, but this is ok for now